.travis.yml

```diff
@@ -9,5 +9,7 @@ install:
 script: bash -ex .travis-docker.sh
 env:
   global:
-    - PACKAGE="xapi-stdext"
-    - PINS="stdext:. xapi-stdext:. xapi-stdext-date:. xapi-stdext-encodings:. xapi-stdext-pervasives:. xapi-stdext-std:. xapi-stdext-threads:. xapi-stdext-unix:. xapi-stdext-zerocheck:."
+    - PINS="stdext:. xapi-stdext-date:. xapi-stdext-encodings:. xapi-stdext-pervasives:. xapi-stdext-std:. xapi-stdext-threads:. xapi-stdext-unix:. xapi-stdext-zerocheck:."
+  jobs:
+    - PACKAGE="stdext"
```

Comment:
Should that be `xapi-stdext`?

Reply:
`xapi-stdext` is a strange compatibility package that was only used used in travis. It was removed from xs-opam last september, in https://github.com/xapi-project/xs-opam/commit/d707ed691f55c1187da8056879e40806c1ffe1d6

We can remove it along stdext after I test a patchset I have removing direct stdext usage from xen-api.